CONTRIBUTING.md

```diff
@@ -175,10 +175,12 @@ bazel test -c opt -k \
 `<package>` can be any package name like `metrics` for example.
 `<py_test_name>` can be any test name given by the `BUILD` file or `*` for all tests of the given package.
 
-### Install in editable mode without compiling
+### Install in editable mode
 
 If you're just modifying Python code (as opposed to C++/CUDA code), 
-then you don't need to use Bazel to run your tests.
+then you don't need to use Bazel to run your tests. 
+And you don't need to compile anything.
+You don't even need a GPU because it's rare that something works for CPU but not for GPU. 
```

Comment:
```suggestion
```
Would just remove this line. Don't love tying ourself to this statement, and since they're not modifying a custom-op it's likely not needed to be said. A counter-example to this is we've found optimizers often have issues when using mixed precision, but they'll be notified of the issue during the CI